code/modules/clothing/suits/ego_gear/waw.dm

```diff
@@ -71,7 +71,7 @@
 	name = "gold rush"
 	desc = "Bare armor. lightweight and ready for combat."
 	icon_state = "gold_rush"
-	armor = list(RED_DAMAGE = 60, WHITE_DAMAGE = 30, BLACK_DAMAGE = 20, PALE_DAMAGE = 0)
+	armor = list(RED_DAMAGE = 70, WHITE_DAMAGE = 40, BLACK_DAMAGE = 30, PALE_DAMAGE = 0)
```

Comment:
And why's that?

Reply:
Hornet right now is 6/3/3/3.
Gold rush was 6/3/2/0. It is just worse hornet. I'm giving it a niche by making it 7/4/2/0

Reply:
Wait a minute, why the hell is it 6/3/3/3? It should be 3/3/3/2.5
![image](https://user-images.githubusercontent.com/53223414/184869111-9ba1e27e-1cfd-4e02-8f4f-3fbde6c349d8.png)

Reply:
Will fix later, for now, this is fine